### PR TITLE
Published icon fix in design card

### LINKED ISSUE #7682: [UI] Designs page design visibility label typo update
#### Current Behavior

[Designs page](https://playground.meshery.io/configuration/designs): designs that have visibility of “Published” are showing with the label of “Public”

#### Desired Behavior

Change the label “Public” into “Published”

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by duckling69**
**Notes for Reviewers**

This PR fixes #7682 

After fix
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Great work @duckling69 ! ✨  

Just couple of points to add on: 

1. Can we make the font text to bold for better readability? 
2. Can we use CSS variables instead of hard-coding color values? 

Rest looks perfect!

**Comment by Ghat0tkach** [comment]
Looks like we are failing some checks ,
/home/runner/work/meshery/meshery/ui/components/MesheryPatterns/MesheryPatternCard.js
  96:1  error  Expected indentation of 14 spaces but found 15  indent

Phew, An indentation can indeed cause failure of checks🥲

File: ui/components/MesheryPatterns/MesheryPatternCard.js [hunk 1]
```diff
@@ -91,7 +91,9 @@ function MesheryPatternCard({
               <Typography style={{ overflow : "hidden", textOverflow : "ellipsis", width : '20rem' }} variant="h6" component="div">
                 {name}
               </Typography>
-              <img  className={classes.img} src={`/static/img/${visibility}.svg`} />
+              <div style={{ border : "1px solid rgb(118 118 118)",color : "rgb(118 118 118)",marginTop : "auto",fontSize : "0.6rem",padding : "0.2rem 0.5rem",borderRadius : "0.2rem" }}>
+                PUBLISHED
```
Comment (leecalcote) [inline ui/components/MesheryPatterns/MesheryPatternCard.js:95]:
```suggestion
               PUBLISHED
```

File: ui/components/MesheryPatterns/MesheryPatternCard.js [hunk 2]
```diff
@@ -91,7 +91,9 @@ function MesheryPatternCard({
               <Typography style={{ overflow : "hidden", textOverflow : "ellipsis", width : '20rem' }} variant="h6" component="div">
                 {name}
               </Typography>
-              <img  className={classes.img} src={`/static/img/${visibility}.svg`} />
+              <div style={{ border : "1px solid rgb(118 118 118)",color : "rgb(118 118 118)",marginTop : "auto",fontSize : "0.6rem",padding : "0.2rem 0.5rem",borderRadius : "0.2rem" }}>
+               PUBLISHED
+               </div>
```
Comment (leecalcote) [inline ui/components/MesheryPatterns/MesheryPatternCard.js:96]:
```suggestion
              </div>
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**